pronotepy/ent/complex_ent.py

```diff
@@ -97,11 +98,11 @@ def ac_reunion(username: str, password: str) -> requests.cookies.RequestsCookieJ
         returns the ent session cookies
     """
     # ENT / PRONOTE required URLs
-    ent_login = "https://portail.college-jeandesme.re:8443/login?service=https:%2F%2Fportail.college-jeandesme.re%2Fpronote%2Feleve.html"
+    ent_login = "https://portail.college-jeandesme.re:8443/login"
 
     # ENT Connection
     session = requests.Session()
-    response = session.get(ent_login, headers=HEADERS)
+    response = session.get(ent_login, headers=HEADERS, verify=False)
```

Comment:
This is very weird... I guess we need it right now, but we should make sure that when they fix their cert, we turn on the verification.

Reply:
Yes you're right